scss/tests/mixins/_utilities-map.test.scss

```diff
@@ -0,0 +1,339 @@
+@mixin before {
+  $utilities: (
+    "flex": (
+      responsive: true,
+      property: flex,
+      values: (fill: 1 1 auto)
+    ),
+    "align": (
+      property: vertical-align,
+      class: align,
+      values:  baseline top
+    ),
+    "margin": (
+      property: margin,
+      class: margin,
+      values: auto
+    )
+  ) !global;
```

Comment:
Missing the values inside a map.

Reply:
I'm not sure what you mean there, sorry. Can you explain? 😊